### PR TITLE
Security review changes

### PR DESCRIPTION
**Pull request by ttheune**
@aren Check out the changes made.  Most concerned with the Cipher changes in sshd_config, but as usual, could use a full review.

### CODE REVIEW

**Comment by aren** [comment]
Do we need to make any changes to sshd_config anymore now that we're not going to do the ForceCommand?

**Comment by ttheune** [comment]
The ciphers and changing allowroot to no.
Can you double check the ciphers?

File: scripts/config.py [hunk 1]
```diff
@@ -160,6 +157,7 @@ def config_vpn(holders,file_list):
     commands = ['xl2tpd','ipsec','fail2ban', 'foxpass-radius-agent']
     call(['/sbin/sysctl','-p'])
     chmod('/etc/ipsec.secrets',0600)
+    chown('/etc/ipsec.secrets',0)
```
Comment (aren) [inline scripts/config.py:168]: add a comment what this does

File: scripts/config.py [hunk 2]
```diff
@@ -25,10 +25,9 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 
-import random,string
 from iptools import IpRange
 from iptools.ipv4 import validate_cidr
-from os import chmod
+from os import chown,chmod,urandom
```
Comment (aren) [inline scripts/config.py]: nit: add spaces after the commas

File: scripts/config.py [hunk 3]
```diff
@@ -68,9 +67,7 @@ def prompt(message, default=None):
         return raw_input( '%s: ' % (message) )
 
 def random_string(len):
-    chars = string.ascii_uppercase + string.digits + string.ascii_lowercase
-    arr = [random.choice(chars) for i in range(len)]
-    return ''.join(arr)
+    return urandom(len).encode('base-64').rstrip()
```
Comment (aren) [inline scripts/config.py]: The problem with base-64 is that the strings end with "=" or "==" much of the time.

I looked into this a bit. The way to do it is the way you did it the first time, but put this line at the top of the function:

random = random.SystemRandom()

it uses urandom under the hood. see here: https://docs.python.org/3/library/random.html

**Pull request merged**